docs/installation.md

```diff
@@ -98,14 +98,15 @@ The cfgov-refresh front end currently uses the following frameworks / tools:
 
 !!! note
     This project requires Node.js v8.5 or higher, and npm v5.2 or higher.
+2. Install [yarn](https://yarnpkg.com/en/docs/install)
```

Comment:
We should include the brew instructions as the recommended installation, just like [we do for autoenv above](https://github.com/cfpb/cfgov-refresh/pull/4346/files#diff-11d8910b58f5c70ccb3bedc0f37c892fR51).